app/javascript/src/LoginPage/LoginPageWrapper.jsx

```diff
@@ -43,40 +39,21 @@ type Props = {
   }
 }
 
-const formModeTuple: [string, string] = ['login', 'password-reset']
-
 type State = {
-  formMode: string,
   loginTitle: string
 }
 
 class SimpleLoginPage extends React.Component<Props, State> {
   constructor (props: Props) {
     super(props)
     this.state = {
-      formMode: 'login',
       loginTitle: 'Log in to your account'
     }
   }
 
-  setFormMode ({win = window}: {win?: Window}) {
-    try {
-      const url = new URL(win.location.href)
-      const formMode = url.search === '?request_password_reset=true' ? formModeTuple[1] : formModeTuple[0]
-      const loginTitle = formMode === 'login' ? 'Log in to your account' : 'Request a password reset link by email'
-      this.setState({formMode, loginTitle})
-    } catch (e) {
-      console.error(e)
-    }
-  }
-
-  componentDidMount () {
-    this.setFormMode(window)
-  }
-
   showForgotCredentials () {
-    const showForgotCredentials = this.state.formMode === formModeTuple[0] && !this.props.disablePasswordReset
-    return showForgotCredentials && <ForgotCredentials providerLoginPath={this.props.providerLoginPath}/>
+    const showForgotCredentials = !this.props.disablePasswordReset
```

Comment:
Here, use destructuring:
```suggestion
    const { disablePasswordReset, providerLoginPath } = this.props
    return !disablePasswordReset && <ForgotCredentials requestPasswordResetPath={providerLoginPath} />
```